lib/charms/layer/jenkins/api.py

```diff
@@ -167,6 +171,13 @@ def get_node_secret(self, node_name):
         except jenkins.JenkinsException:
             return False
 
+    def set_update_center(self, url=None):
+        """Set the update center or reset it to default"""
+        url = url or "https://updates.jenkins.io/stable/update-center.json"
```

Comment:
I would definitely add a debug line here so we know which update_center has been picked.

Reply:
d3a6a39